### PR TITLE
Add support for RED FMTP

### LINKED ISSUE #14: New Chromium sends opus+red FMTP, breaks SDP parsing
As of the latest Chromium, this attribute is showing up in the SDP counter-offer from the browser:

`a=fmtp:63 111/111`

This gets caught in the fall-through case of `parse_param` in `ExSDP.Attribute.FMTP` and results in `{:error, :unsupported_parameter}`.

https://groups.google.com/g/discuss-webrtc/c/5761etCrSuA

### PR DESCRIPTION
**Pull request by mickel8**
closes #14

### CODE REVIEW

File: lib/ex_sdp/attribute/fmtp.ex [hunk 1]
```diff
@@ -173,20 +194,74 @@ defmodule ExSDP.Attribute.FMTP do
...
+  end
+
+  defp validate_dtmf_tones(dtmf_tones) do
+    Enum.all?(dtmf_tones, fn dtmf_tone ->
+      case String.split(dtmf_tone, "-") do
+        [start_range, end_range] ->
+          with {:ok, start_range} <- Utils.parse_numeric_string(start_range),
+               {:ok, end_range} <- Utils.parse_numeric_string(end_range),
+               true <- start_range < end_range and 0 <= start_range and end_range <= 255 do
+            true
```
Comment (Rados13) [inline lib/ex_sdp/attribute/fmtp.ex]: Shouldn't we put this condition inside do block?

File: lib/ex_sdp/attribute/fmtp.ex [hunk 2]
```diff
@@ -173,20 +194,74 @@ defmodule ExSDP.Attribute.FMTP do
...
+          else
+            _error -> false
+          end
+
+        [single_tone] ->
+          with {:ok, single_tone} <- Utils.parse_numeric_string(single_tone),
+               true <- 0 <= single_tone and single_tone <= 255 do
```
Comment (Rados13) [inline lib/ex_sdp/attribute/fmtp.ex]: Shouldn't we put this condition also inside do block?

File: lib/ex_sdp/attribute/fmtp.ex [hunk 3]
```diff
@@ -2,10 +2,19 @@ defmodule ExSDP.Attribute.FMTP do
...
   This module represents fmtp (RFC 5576).
 
-  Parameters for H264 (not all, RFC 6184), VP8, VP9 and OPUS (RFC 7587) are currently supported.
+  Parameters for:
+  * H264 (not all, RFC 6184),
+  * VP8, VP9, OPUS (RFC 7587)
+  * RTX (RFC 4588)
+  * FLEXFEC (RFC 8627)
+  * Telephone Events (RFC 4733)
+  * RED (RFC 2198)
```
Comment (Rados13) [inline lib/ex_sdp/attribute/fmtp.ex:11]: RED has an additional parameter `forwardshift` described in [RFC 6354](https://www.rfc-editor.org/rfc/pdfrfc/rfc6354.txt.pdf). Shouldn't we add it also, as we adding support for RED? The other way change the description to look the same as for H264.

Comment (mickel8) [reply lib/ex_sdp/attribute/fmtp.ex:11]: As far as I understand this, `fwdred` is another Media Type so it is something equivalent to `red` -- it is not parameter for `red`.

**Review by Rados13 — APPROVED** [review]

**Pull request merged**